testgen/generators/common.js

```diff
@@ -7,10 +15,32 @@ function sample_tests(all_tests, run_limit) {
   }
 
   let size_all = all_tests.length;
-  let increment = Math.floor(size_all / run_limit);
-  let samples = [];
-  for (let index = 0; index < size_all; index += increment) {
-    samples.push(all_tests[index]);
+  label_increment = Math.floor(size_all / run_limit);
+  max_count = run_limit;
+  added_count = 0;
+
+  if (debug) {
```

Comment:
I filed https://github.com/unicode-org/conformance/issues/338 to capture the idea of improving & simplifying our JS logging code by using a library.

Reply:
Noted! What about https://github.com/pimterry/loglevel?

Reply:
I removed the debugging code.